### PR TITLE
Translation edits

### PR DESCRIPTION
**Pull request by antonellanapolitano**

### CODE REVIEW

File: lib/views/help/privacy.it.html.erb [hunk 1]
```diff
@@ -59,33 +59,18 @@ Ma chiedilo solo se è davvero necessario: purtroppo non abbiamo la possibilità
...
-Per questo motivo non cancelliamo mai le richieste e relative risposte.  
+<p>La piattaforma è un archivio pubblico e permanente delle richieste di accesso. 
+Anche se ricevi una risposta che non ti serve più, ricorda che potrebbe essere utile a qualcun'altro. 
+Per questo motivo non cancelliamo mai richieste e relative risposte.  
 
 </p>
 
 <p>In casi eccezionali possiamo rimuovere o modificare il tuo nome (<a href="#takedown">vedi prossima domanda</a>) e, analogamente, possiamo fare un'eccezione per altre informazioni personali.  
-Se sei preoccupato per questo, guarda la sezione sugli <a href="#real_name">pseudonyms</a>. 
+Se sei preoccupato per questo, guarda la sezione sugli <a href="#real_name">pseudonimi</a>.
```
Comment (garethrees) [inline lib/views/help/privacy.it.html.erb]: We need to use the English anchor here, as per #49

File: lib/views/help/privacy.it.html.erb [hunk 2]
```diff
@@ -10,21 +10,21 @@
...
 
 <dd><p>Non mostreremo il tuo indirizzo a nessuno, a meno che non via sia un obbligo di legge al riguardo. 
-Questo vale anche per la pubblica amministrazione a cui stai inviando la richiesta. Vedranno solo un indirizzo email che termina con @chiediamo.org, che sarà specifico per quella richiesta.
+Questo vale anche per la pubblica amministrazione a cui stai inviando la richiesta. Vedranno solo un indirizzo email che termina con @chiedi.dirittodisapere.it, che sarà specifico per quella richiesta.
   </p>
 <p>Se spedisci un messaggio a un altro utente del sito, allora il tuo indirizzo verrà mostrato (e prima ti verrà segnalato).</p>
 </dd>
 
-<dt id="spam">Inizierete a inondarmi con il solito, fastidiosissimo spam <a href="#spam">#</a> </dt>
+<dt id="spam">Inizierete a inondarmi con il solito, fastidiosissimo spam <a href="#spam">#</a>... </dt>
```
Comment (garethrees) [inline lib/views/help/privacy.it.html.erb]: Could use `&hellip;` here to get a proper ellipsis (`…` vs `...`)

File: lib/views/help/privacy.it.html.erb (hunk 1 above)
Comment (antonellanapolitano) [inline lib/views/help/privacy.it.html.erb]: ok

Il giorno 16 febbraio 2015 12:29, Gareth Rees <notifications@github.com> ha
scritto:

> In lib/views/help/privacy.it.html.erb
> <https://github.com/mysociety/dirittodisapere-theme/pull/63#discussion_r24745240>
> :
>
> >
> >  </p>
> >
> >  <p>In casi eccezionali possiamo rimuovere o modificare il tuo nome (<a href="#takedown">vedi prossima domanda</a>) e, analogamente, possiamo fare un'eccezione per altre informazioni personali.
> > -Se sei preoccupato per questo, guarda la sezione sugli <a href="#real_name">pseudonyms</a>.
> > +Se sei preoccupato per questo, guarda la sezione sugli <a href="#real_name">pseudonimi</a>.
>
> We need to use the English anchor here, as per #49
> <https://github.com/mysociety/dirittodisapere-theme/pull/49>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mysociety/dirittodisapere-theme/pull/63/files#r24745240>
> .
>



-- 



-----------------------------------------------------------------------

Antonella Napolitano
Cell. 339/8182993
LinkedIn:  <http://www.linkedin.com/in/antonellanapolitano>
http://www.linkedin.com/in/antonellanapolitano
Blog:  <http://svaroschi.blogspot.com/>http://svaroschi.blogspot.com
Twitter:  <http://twitter.com/svaroschi>http://twitter.com/svaroschi

Make a loan on Kiva <http://www.kiva.org/invitedby/svaroschi> - A place for
piloting innovations in person-to-person lending

File: lib/views/help/privacy.it.html.erb [hunk 3]
```diff
@@ -59,33 +59,18 @@ Ma chiedilo solo se è davvero necessario: purtroppo non abbiamo la possibilità
...
 
-<dt id="full_address">Mi hanno chiesto l'indirizzo di casa! <a href="#full_address">#</a> </dt>
+<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome? <a href="#delete_requests">#</a> </dt>
 
 <dd>
-<p>Verificare la legge italiana</p>
 
-</dd>
-
-<dt id="postal_answer">No, no, vogliono l'indirizzo per spedirmi una risposta cartacea! <a href="#postal_answer">#</a> </dt>
```
Comment (garethrees) [inline lib/views/help/privacy.it.html.erb]: This section is required as it's linked to from elsewhere on the site.

File: lib/views/help/privacy.it.html.erb [hunk 4]
```diff
@@ -59,33 +59,18 @@ Ma chiedilo solo se è davvero necessario: purtroppo non abbiamo la possibilità
...
-
-<dd>
-<p>verificare legge italiana</p>
-
-
-</dd>
-
-<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome?  <a href="#delete_requests">#</a> </dt>
```
Comment (garethrees) [inline lib/views/help/privacy.it.html.erb]: This section is required as it's linked to from elsewhere on the site.

Comment (antonellanapolitano) [inline lib/views/help/privacy.it.html.erb]: Hi Gareth,

I submitted another pull request explaining the changes.

2015-02-16 12:35 GMT+01:00 Gareth Rees <notifications@github.com>:

> In lib/views/help/privacy.it.html.erb
> <https://github.com/mysociety/dirittodisapere-theme/pull/63#discussion_r24745501>
> :
>
> >
> > -</dd>
> > -
> > -<dt id="postal_answer">No, no, vogliono l'indirizzo per spedirmi una risposta cartacea! <a href="#postal_answer">#</a> </dt>
> > -
> > -<dd>
> > -<p>verificare legge italiana</p>
> > -
> > -
> > -</dd>
> > -
> > -<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome?  <a href="#delete_requests">#</a> </dt>
>
> This section is required as it's linked to from elsewhere on the site.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mysociety/dirittodisapere-theme/pull/63/files#r24745501>
> .
>



-- 



-----------------------------------------------------------------------

Antonella Napolitano
Cell. 339/8182993
LinkedIn:  <http://www.linkedin.com/in/antonellanapolitano>
http://www.linkedin.com/in/antonellanapolitano
Blog:  <http://svaroschi.blogspot.com/>http://svaroschi.blogspot.com
Twitter:  <http://twitter.com/svaroschi>http://twitter.com/svaroschi

Make a loan on Kiva <http://www.kiva.org/invitedby/svaroschi> - A place for
piloting innovations in person-to-person lending

**Pull request closed without merging**